CryptoPrices/Sources/Data/Tests/RepositoriesTests/CoinRepository/CoinRepositorySpec.swift

```diff
@@ -0,0 +1,55 @@
+import DataTestHelpers
+import NetworkCore
+import Nimble
+import Quick
+import TestHelpers
+@testable import Repositories
+
+final class CoinRepositorySpec: QuickSpec {
+
+    override func spec() {
+        var coinRepository: CoinRepository!
+        var coinAPI: MockCoinAPI!
+
+        describe("The CoinRepository") {
```

Comment:
The capitalization of the word `The` might be unnecessary here 🙏
```suggestion
        describe("the CoinRepository") {
```

Reply:
Updated with [#062134e](https://github.com/nimblehq/nimble-crypto-ios/pull/33/commits/062134eefb9111754b288ed6614af17451aa4eed)!